lndr-backend/src/Lndr/Db/VerifiedCredits.hs

```diff
@@ -42,8 +42,8 @@ deleteExpiredSettlementsAndAssociatedCredits conn = do
     commit conn
 
 
-settlementCreditsToVerify :: Connection -> IO [Text]
-settlementCreditsToVerify conn = fmap fromOnly <$> query_ conn "SELECT hash FROM settlements WHERE tx_hash IS NOT NULL AND verified = FALSE"
+txHashesToVerify :: Connection -> IO [Text]
+txHashesToVerify conn = fmap fromOnly <$> query_ conn "SELECT tx_hash FROM settlements WHERE tx_hash IS NOT NULL AND verified = FALSE"
```

Comment:
you still want `GROUP BY` here, don't you? otherwise you'll get duplicate txHashes when a multisettlement is pending.